### PR TITLE
1. remove unused using, 2. Reorganized the using, 3. Moved the class into separate file and 4. Added line and spacing formats

### PR DESCRIPTION
**Pull request by vishipayyallore**

### CODE REVIEW

**Comment by vishipayyallore** [comment]
@sughosneo 
Good afternoon. Please look into this whenever you get a couple of minutes. Many thanks.

Regards,
Swamy

**Comment by sughosneo** [comment]
Hi @vishipayyallore , thank you for submitting this PR. Will test and update.

File: src/Services/Catalog/Catalog.API/Proto/catalog.proto [hunk 1]
```diff
@@ -1,60 +1,69 @@
...
-    double price=4;
-    string picture_file_name=5;
-    string picture_uri=6;
-    CatalogType catalog_type=8;
-    CatalogBrand catalog_brand=10;
-    int32 available_stock=11;
-    int32 restock_threshold=12;
-    int32 max_stock_threshold=13;
-    bool on_reorder=14;
+
```
Comment (sughosneo) [inline src/Services/Catalog/Catalog.API/Proto/catalog.proto:21]: I would personally just avoid these extra line break.

File: src/Services/Catalog/Catalog.API/IntegrationEvents/Events/OrderStockItem.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.eShopOnContainers.Services.Catalog.API.IntegrationEvents.Events
```
Comment (sughosneo) [inline src/Services/Catalog/Catalog.API/IntegrationEvents/Events/OrderStockItem.cs:1]: I wouldn't recommend to create a separate class file for class `OrderStockItem` for the following reason. 
- It's specific to `OrderStockConfirmedIntegrationEvent`
- And the `Events` directory contains all events. Keeping a public class as a separate file might not make much sense.

**Comment by sughosneo** [comment]
Hi @vishipayyallore - I have included few suggestions. If you could review those. 

Thank you.

File: src/Services/Catalog/Catalog.API/IntegrationEvents/Events/OrderStockItem.cs (hunk 2 above)
Comment (vishipayyallore) [reply src/Services/Catalog/Catalog.API/IntegrationEvents/Events/OrderStockItem.cs:1]: @sughosneo 
Sure let me address those and ping you. Many thanks.

**Comment by vishipayyallore** [comment]
> Hi @vishipayyallore - I have included few suggestions. If you could review those.
> 
> Thank you.

@sughosneo 
I have reverted the commit. I will exclude those two suggestions and I will check in the code. Many thanks.

**Comment by vishipayyallore** [comment]
> > Hi @vishipayyallore - I have included few suggestions. If you could review those.
> > Thank you.
> 
> @sughosneo
> I have reverted the commit. I will exclude those two suggestions and I will check in the code. Many thanks.

@sughosneo (Sumit),
Good afternoon. I have addressed those comments. Could you please review it? I will create another PR after this PR gets merged. Many thanks.

Regards,
Swamy

**Comment by sughosneo** [comment]
- Reviewed the changes for the following project 
`Basket.API`,`Catalog.API`,`Webhooks.API`,`Application.FunctionalTests`
- Tested the changes with `docker-compose`. LGTM

**Review by sughosneo — APPROVED** [review]

**Pull request merged**